quickstart/resources/views/category/create.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('layout.master')
+@section('content')
+@component('layout.components.title')
+{{__('Create page')}}
+@endcomponent
+<div class="container mt-3">
+  <div class="d-flex flex-row">
+    <div class="col-12 px-0">
+      <form action="{{route('category.store')}}" method="post">
+        <input type="hidden" name="_token" value="{{csrf_token()}}">
+        <div class="form-group ">
+          <label class="text-uppercase font-weight-bold" for="name">{{__('Name')}}</label>
+          <input type="text" class="form-control rounded-0" id="name" placeholder="{{__('Name')}}" name="name">
+        </div>
+        <div class="form-group ">
+            <label for="parent" class="text-uppercase font-weight-bold">{{__('Parent')}}:</label>
+            <select class="form-control" name="parent" id="">
+                <option value="0">----ROOT----</option>
+                {{ GetCategory($categories,0,'',0) }}
+            </select>
+        </div>
+        <div class="form-group ">
+          <button type="submit" class="btn btn-danger text-uppercase rounded-0 font-weight-bold">
+            {{__('Confirm')}}
```

Comment:
```suggestion
            {{ __('Confirm') }}
```
fix tương tự các chỗ khác nhé